packages/terra-graphs/CHANGELOG.md

```diff
@@ -12,4 +12,7 @@
   * Created component and wdio test cases for the Pie chart.
   * Created component and wdio test cases for the Combination graph.
 
+* Changed
+  * Fixed the goal line element in bar graph overlays the ability to click on the bar data.
```

Comment:
nit: `Fixed bug causing the goal line element to prevent clicking on the bar graph underneath.`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/153/commits/026c5cc2a95a48b2a0329922394cb583c8aa44ab#diff-d24190ff244a9f2416eeb3cbe315e41345dab81295d0fcccd748cd7f6c93a84aR16).